src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -2468,6 +2493,32 @@ components:
           description: A unique reference for a FutureWorkInterests resource (if it already exists).
           example: c88a6c48-97e2-4c04-93b5-98619966447b
 
+    SubjectAccessRequestContent:
+      properties:
+        induction:
+          $ref: '#/components/schemas/InductionResponse'
+        goals:
+          uniqueItems: true
+          type: array
+          items:
+            $ref: '#/components/schemas/GoalResponse'
+
+    SarSuccessResponse:
+      properties:
+        content:
+          $ref: '#/components/schemas/SubjectAccessRequestContent'
+
+    SarErrorResponse:
```

Comment:
I think we already have the equivalent of this type, in that we already have `ErrorResponse` in this spec and is already a generated class.
I dont think we need this ... 🤔